lms/djangoapps/lms_xblock/mixin.py

```diff
@@ -177,6 +180,9 @@ def _has_nonsensical_access_settings(self):
         else:
             return False
 
+    def _is_unit(self):
+        return get_parent_unit(self) is None and self.get_parent()
```

Comment:
A comment explaining the logic would be helpful here. It's not clear to me why this combination means that it is a unit.